docs/installation-guide/quickstart.rst

```diff
@@ -0,0 +1,345 @@
+
+   .. code-block:: shell
+
+    $ ssh-keygen
+    $ ssh-copy-id <IP-of-the-bootstrap-node>
+
+3. Copy the private key in the pki folder of MetalK8s. This will be use further for
+   adding new nodes on the cluster.
+
+   .. code-block:: shell
+
+    $ mkdir -p /etc/metalk8s/pki/
+    $ cp /root/.ssh/id_rsa /etc/metalk8s/pki/id_rsa
+
+Install the bootstrap node
+--------------------------
+1. Run the script to install the bootstrap node.
+
+   .. code-block:: shell
+
+    $ /srv/scality/metalk8s-2.0/bootstrap.sh
+
+2. You need to install kubectl for the CLI and check if pods are running well.
+
+   .. code-block:: shell
+
+    $ yum install —disablerepo=* —enablerepo=metalk8s-* kubectl
+    $ export KUBECONFIG=/etc/kubernetes/admin.conf
+
+    $ kubectl get node
+    NAME                   STATUS    ROLES                         AGE       VERSION
+    bootstrap              Ready     bootstrap,etcd,infra,master   17m       v1.11.7
+
+    $ kubectl get pods --all-namespaces -o wide
+    NAMESPACE     NAME                                          READY     STATUS    RESTARTS   AGE       IP             NODE                  NOMINATED NODE
+    kube-system   calico-node-zw74v                             1/1       Running   0          18m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   coredns-6b9cb79bf4-jbtxc                      1/1       Running   0          18m       10.233.0.2     bootstrap.novalocal   <none>
+    kube-system   coredns-6b9cb79bf4-tdmz8                      1/1       Running   0          18m       10.233.0.4     bootstrap.novalocal   <none>
+    kube-system   etcd-bootstrap                                1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-apiserver-bootstrap                      1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-controller-manager-bootstrap             1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-proxy-mwxhf                              1/1       Running   0          18m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-scheduler-bootstrap                      1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   metalk8s-ui-656f6857b-cdt5p                   1/1       Running   0          18m       10.233.0.3     bootstrap.novalocal   <none>
+    kube-system   package-repositories-bootstrap                1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   registry-bootstrap                            1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   salt-master-bootstrap                         2/2       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+
+
+Adding a master to the cluster
```

Comment:
Can we use the wording "control-plane node"/"workload-plane node" instead of just "master"/"node"?

Reply:
See #1123